libnavigation-base/api/current.txt

```diff
@@ -1341,6 +1341,7 @@ package com.mapbox.navigation.base.trip.model.roadobject.incident {
   }
 
   public final class IncidentInfo {
+    method public java.util.List<java.lang.String>? getAffectedRoadNames();
```

Comment:
Is that intentional? I don't see any changes in this module.

Reply:
https://github.com/mapbox/mapbox-navigation-android/pull/6032#discussion_r916686793
It seems like CI didn't catch it in the other PR. This should have been added there.